MapboxAndroidDemo/src/global/java/com/mapbox/mapboxandroiddemo/MainActivity.java

```diff
@@ -897,6 +898,14 @@ private void initializeModels() {
       null,
       R.string.activity_camera_slowly_rotating_url, false, BuildConfig.MIN_SDK_VERSION));
 
+    exampleItemModels.add(new ExampleItemModel(
+      R.id.nav_camera,
+      R.string.activity_camera_zoom_to_cluster_leaves_title,
+      R.string.activity_camera_zoom_to_cluster_leaves_description,
+      new Intent(MainActivity.this, ZoomToShowClusterLeavesActivity.class),
+      null,
+      R.string.activity_camera_zoom_to_cluster_leaves_url, false, BuildConfig.MIN_SDK_VERSION));
```

Comment:
Not super crucial, but since the layout file was renamed to activity_zoom_to_show_cluster_leaves.xml, I think it would be best if the _show addition was propagated to these R.strings as well.
```suggestion
      R.string.activity_camera_zoom_to_show_cluster_leaves_title,
      R.string.activity_camera_zoom_to_show_cluster_leaves_description,
      new Intent(MainActivity.this, ZoomToShowClusterLeavesActivity.class),
      null,
      R.string.activity_camera_zoom_to_show_cluster_leaves_url, false, BuildConfig.MIN_SDK_VERSION));
```